src/pages/Actions/ActionEvent/EventCreate/EventCreate.tsx

```diff
@@ -259,475 +261,484 @@ export default function ({ onCreate, setShowEventCreateDrawer }: Props) {
     setShowEventCreateDrawer(false);
   };
 
+  const handleClose = () => {
+    setShowEventCreateDrawer(false);
+  };
+
   function warning() {
     notificationLogic("warning", "Спочатку оберіть тип події.");
   }
 
   return (
-    <Form
-      name="basic"
-      form={form}
-      onFinish={handleFinish}
-      id="area"
-      style={{ position: "relative" }}
-    >
-      <Row justify="start" gutter={[12, 0]}>
-        <Col md={24} xs={24}>
-          <Form.Item
-            label="Тип події"
-            name="EventTypeID"
-            rules={[{ required: true, message: isNotChosen("Тип події") }]}
-            className={classes.radio}
-          >
-            <Radio.Group
-              buttonStyle="solid"
-              className={classes.eventTypeGroup}
-              onChange={onChange}
-              value={categories}
+    <>
+      <ButtonCollapse handleClose={handleClose} />
+      <Form
+        name="basic"
+        form={form}
+        onFinish={handleFinish}
+        id="area"
+        style={{ position: "relative" }}
```

Comment:
It is better not to use inline styles to avoid "spaghetti code".

Reply:
thank you for your suggestion